Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9
+
+ADD requirements.txt /tmp/requirements.txt
+
+RUN pip install -r /tmp/requirements.txt
+
+ADD src /var/server/src
```

Comment:
Don't we want COPY instead of ADD? ADD does tar extraction out of the box, might be error prone

Reply:
Changed